x/simulation/simulate.go

```diff
@@ -138,10 +138,12 @@ func SimulateFromSeed(
 	}
 
 	// set exported params to the initial state
-	exportedParams = params
+	if exportParams && exportParamsHeight == 0 {
+		exportedParams = params
+	}
 
 	// TODO: split up the contents of this for loop into new functions
-	for height := 1; height <= numBlocks && !stopEarly; height++ {
+	for height := initialHeight; height < numBlocks+initialHeight && !stopEarly; height++ {
```

Comment:
Why `numBlocks+initialHeight`? If I run simulation with `NumBlocks=500` and halt it at after block 200 and then resume it via `InitialBlockHeight=201` it should still resume to finish at block 500, not 701 right? Or am I just being silly?

Reply:
updated `NumBlocks` description